### PR TITLE
Feature/enable appveyor

### PR DESCRIPTION
**Pull request by drnlm**
This adds appveyor support for testing pygame_cffi on windows.

A number of tests faill, at least partly because RWops isn't correct on windows platforms, but at least the tests run.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+# Appveyor's build step is specific to .NET projects, so we build in the
+# install step instead.
+build: false
+
+test_script:
+  - "python -m test"
+
+#artifacts:
+#  - path: dist\*
```
Comment (hodgestar) [inline appveyor.yml]: Should this be uncommented? Or removed?

File: appveyor/README.rst [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Appveyor is a service we use to build windows binaries for pygame_cffi.
+
+We borrow varioys helpers, like the install.psi script, from the pygame project.
```
Comment (hodgestar) [inline appveyor/README.rst]: s/psi/ps1/?

File: appveyor/copydlls.ps1 [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        $target = "x64"
+    }
+    $prebuilt_dir = "prebuilt-"+$target
+
+    $basedir = $pwd.Path + "\"
+    $dlls = $basedir + $prebuilt_dir + "\lib\" + "*.dll"
+    $destpath = $basedir + "pygame"
+
+    Write-Host "dlls from" $dlls
+    Write-Host "destiation" $destpath
```
Comment (hodgestar) [inline appveyor/copydlls.ps1]: s/destiation/destination/

File: appveyor/install.ps1 [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    $prebuilt_zip = "prebuilt-" + $target + ".zip"
+
+    $basedir = $pwd.Path + "\"
+    $filepath = $basedir + $prebuilt_zip
+    if (Test-Path $filepath) {
+        Write-Host "Reusing" $filepath
+        return $filepath
+    }
+
+    # Download and retry up to 5 times in case of network transient errors.
```
Comment (hodgestar) [inline appveyor/install.ps1]: s/5/3/?

File: helpers/__init__.py [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+def get_inc_dir():
+    if sys.platform.startswith("win"):
+        if platform.architecture()[0] == '32bit':
+            # 32 bit
+            return ['prebuilt-x86/include', 'prebuilt-x86/include/SDL']
+        else:
+            return ['prebuilt-x64/include', 'prebuilt-x64/include/SDL']
+    return ['/usr/include', '/usr/include/SDL', '/usr/local/include/SDL']
+
```
Comment (hodgestar) [inline helpers/__init__.py]: I think we might be missing a newline between these two functions. Should we perhaps add docstrings to these?

File: appveyor.yml (hunk 1 above)
Comment (drnlm) [inline appveyor.yml]: My plan is to have appveyor build wheels for us. It already runs bdist_wheel, but I still need to work out how to include the required dlls. Once that's in place, I'll uncomment this so we can grab the produced wheels.

File: appveyor/install.ps1 (hunk 4 above)
Comment (drnlm) [inline appveyor/install.ps1]: We copy the install.ps1 script verbatim from pygame, comment errors and all.

I'd prefer to keep the two in sync, so it's trivial to spot when pygame updates something.

Comment (hodgestar) [inline appveyor/install.ps1]: Cool.

File: README.md [hunk 6]
```diff
@@ -48,3 +50,18 @@ version 1.3.0).
    argument.
 * Conformance between pygame and pygame_cffi: See `conformance/README`
 * pygame_cffi functionality example apps are in the `demos` directory
+
+
+## Building on windows
+
+To build on windows, you need to download the appropriate dependancy libraries.
```
Comment (hodgestar) [inline README.md]: s/dependancy/dependency/

File: appveyor.yml (hunk 1 above)
Comment (hodgestar) [inline appveyor.yml]: Cool.

**Pull request merged**